File: patterns/aufgeschobene-automatisierte-aktionen/rules-python/aufgeschobene_automatisierte_aktionen_example.py
from openhab import rule, Registry
from openhab.triggers import ItemStateChangeTrigger


@rule(triggers=[ItemStateChangeTrigger("DayNight", state="DAY", previous_state="NIGHT")])
class RolladenNachSonnenaufgangOeffnen:
    def execute(self, module, input):
        Registry.getItem("LoungeBlind_Timer").sendCommand("+15m->UP")

File: patterns/aufgeschobene-automatisierte-aktionen/rules-js/aufgeschobene-automatisierte-aktionen-example.js
rules.JSRule({
  name: "Rollladen nach Sonnenaufgang oeffnen",
  triggers: [triggers.ItemStateChangeTrigger("DayNight", "NIGHT", "DAY")],
  execute: (event) => {
    items.getItem("LoungeBlind_Timer").sendCommand("+15m->UP");
  }
});
